### PR TITLE
Revamp Grafana Flightsql install instructions

### PR DESCRIPTION
**Pull request by sanderson**
With the FlightSQL plugin now officially supported, installation instructions have gotten a lot simpler. This PR updates the Grafana doc in Cloud IOx with a new flow that relies on the streamlined plugin installation paths provided by Grafana.

- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-iox/query-data/tools/grafana.md [hunk 1]
```diff
@@ -29,170 +30,82 @@ Install the [grafana-flight-sql-plugin](https://github.com/influxdata/grafana-fl
...
 - [Build visualizations with Grafana](#build-visualizations-with-grafana)
 
 <!-- /TOC -->
 
-## Install Grafana
+## Install Grafana or login to Grafana Cloud
 
-Follow [Grafana instructions](https://grafana.com/docs/grafana/latest/setup-grafana/installation/)
-to Install Grafana for your operating system.
+If user the open source version of **Grafana**, follow the
```
Comment (jstirnaman) [inline content/influxdb/cloud-iox/query-data/tools/grafana.md:43]:
```suggestion
If using the open source version of **Grafana**, follow the
```

File: content/influxdb/cloud-iox/query-data/tools/grafana.md [hunk 2]
```diff
@@ -29,170 +30,82 @@ Install the [grafana-flight-sql-plugin](https://github.com/influxdata/grafana-fl
...
+<!---------------------------- BEGIN GRAFANA CLOUD ---------------------------->
 
--  Mount the plugin directory (`/custom/plugins/directory/influxdata-flightsql-datasource`) as a volume to your Grafana container.
+1. In your Grafana Cloud instance, navigate to **Administration** > **Plugins**.
+2. Search for and select the **FlightSQL** plugin.
+3. Click **Install via grafana.com** to navigate to the plugin page.
+4. On the plugin page, click **Install plugin**.
 
-### Install with Docker Run
+After a moment, Grafana Cloud will complete the plugin installation in your
```
Comment (jstirnaman) [inline content/influxdb/cloud-iox/query-data/tools/grafana.md:91]:
```suggestion
After a moment, Grafana Cloud completes the plugin installation in your
```

**Review by jstirnaman — APPROVED** [review]
LGTM - some minor fixes.

**Review by helenosheaa — APPROVED** [review]
nice!

**Pull request merged**